pypeit/io.py

```diff
@@ -657,6 +658,10 @@ def write_to_hdu(d, name=None, hdr=None, force_to_bintbl=False):
             types.
 
     """
+    # Silence the "Keyword name ... is greater than 8 characters ... a HIERARCH
+    #   card will be created" Warnings from [astropy.io.fits.card]
+    warnings.simplefilter('ignore', VerifyWarning)
```

Comment:
Do we need the `from astropy.io.fits.verify import VerifyWarning` import?  Can we instead set this to `warnings.simplefilter('ignore', fits.verify.VerifyWarning)` since we already have `fits` imported?

Reply:
@kbwestfall I'll fix this as part of my forthcoming PR related to users specifying local files for, _e.g._, `redi_arxiv`.